tests-bundle/README.md

```diff
@@ -0,0 +1,30 @@
+# Tests Bundle
+
+This folder contains the full bundle tests for Kubeflow. These tests are designed to deploy Kubeflow and run various tests on the deployed Kubeflow, including Selenium tests for interacting with the Kubeflow UI. The long-term vision for this bundle is to add hundreds of tests to thoroughly test the entire Kubeflow bundle before deployment.
+
+## Test Structure
+
+The tests are organized into directories based on the version of Kubeflow. Each version has its own suite of tests, although there may be similarities between versions. Minimally, each version's directory contains a Python file for deploying Kubeflow and one or more other files representing other tests.
+
+The test order is enforced to ensure that the deployment test runs and succeeds before further tests can run. This order is defined with the help of pytest markers, where the "deploy" tests are executed first.
+
+## How to Run the Tests as a GitHub Action
+
+These steps assume that you are familiar with GitHub Actions and have the necessary permissions to trigger and run actions in the repository.
+
+To run the tests as a GitHub Action, follow these steps:
+
+1. Trigger the full bundle tests [GitHub action](/.github/workflows/full-bundle-tests.yaml) by selecting `View Runs -> Run workflow`.
```

Comment:
isn't it?

https://github.com/canonical/bundle-kubeflow/actions/workflows/full-bundle-tests.yaml

Reply:
I think I found a better way that solves both problems:

- Independent of the repo URL
- Links directly to the action (and not the YAML file)

Reply:
This seems to work OK in general. Although one potential issue is if GitHub changes their URL structure. I have examples of it working here: https://github.com/ColmBhandal/readmeabsolutelinks.